home-assistant-voice.yaml

```diff
@@ -1354,11 +1354,23 @@ script:
   # Why is this wrapped on a script?
   #   Becasue we want to stop the sequence if the TTS step is faster than that.
   #   This allows us to prevent having the deactivation of the stop word before its own activation.
-  - id: activate_stop_word_if_tts_step_is_long
+  - id: activate_stop_word_once
     then:
       - delay: 1s
       # Enable stop wake word
-      - lambda: id(stop).enable();
+      - if:
+          condition:
+            switch.is_off: timer_ringing
+          then:
+            - lambda: id(stop).enable();
+            - wait_until:
+                not:
+                  media_player.is_announcing:
+            - if:
+                condition:
```

Comment:
should this be in the wait_until? Now if this condition doesn't pass, the stop wake word would never be disabled.